core/src/main/java/com/predic8/membrane/core/interceptor/apikey/ApiKeysInterceptor.java

```diff
@@ -123,6 +127,11 @@ public Optional<LocationNameValue> getKey(Exchange exc) {
                          .findFirst();
     }
 
+    /**
+     * @description Controls whether API key validation is enforced or optional. Optional will still load scopes into Exchange property.
+     * @default true
+     * @example false
+     */
```

Comment:
Controls whether API key validation is enforced or optional. Optional will still load scopes and makes them available in the Exchange property XXXX.